Python/libraries/recognizers-date-time/recognizers_date_time/date_time/utilities.py

```diff
@@ -0,0 +1,405 @@
+from enum import Enum, IntEnum, IntFlag
+from abc import ABC, abstractmethod
+from typing import List, Dict, Pattern, Union
+from datetime import datetime, timedelta
+import calendar
+from datedelta import datedelta
+import regex
+
+from recognizers_text.extractor import ExtractResult
+from recognizers_text.utilities import RegExpUtility
+from recognizers_date_time.date_time.constants import TimeTypeConstants
+from recognizers_date_time.date_time.extractors import DateTimeExtractor
+from recognizers_date_time.date_time.parsers import DateTimeParser, DateTimeParseResult
+
+class DateTimeOptions(IntFlag):
+    NONE = 0
+    SKIP_FROM_TO_MERGE = 1
+    SPLIT_DATE_AND_TIME = 2
+    CALENDAR = 4
```

Comment:
This PR is based on the current .JS version, right? What's the current plan to cover the delta from .NET?
For example, .NET has two more options and corresponding code. The Java version is starting from .NET and will cover those.

Reply:
This version is based on a previous JS code. We think it will be easier to merge this pull-request first and later fix the current gap between implementations